Userland/Utilities/mktemp.cpp

```diff
@@ -78,8 +78,10 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
             target_directory = resolved_path.dirname();
             file_template = resolved_path.basename();
         } else {
-            char const* env_directory = getenv("TMPDIR");
-            target_directory = env_directory && *env_directory ? StringView { env_directory, strlen(env_directory) } : "/tmp"sv;
+            target_directory = "/tmp"sv;
+            auto const* env_directory = getenv("TMPDIR");
+            if (*env_directory != 0 && env_directory != nullptr)
```

Comment:
This doesn't look right. We're dereferencing `env_directory` *before* checking if it is null :)

Reply:
I've just learnt something! Fixed now!